authors/settings.py

```diff
@@ -133,11 +133,15 @@
 # called `INSTALLED_APPS`.
 AUTH_USER_MODEL = 'authentication.User'
 
-REST_FRAMEWORK = {
-    'EXCEPTION_HANDLER': 'authors.apps.core.exceptions.core_exception_handler',
-    'NON_FIELD_ERRORS_KEY': 'error',
 
+REST_FRAMEWORK = {
+    'DEFAULT_PERMISSION_CLASSES': (
+        'rest_framework.permissions.IsAuthenticated',
+    ),
     'DEFAULT_AUTHENTICATION_CLASSES': (
-        'authors.apps.authentication.backends.JWTAuthentication',
```

Comment:
Isn't this change out of the scope of this task? in that case we need to add a new task on Pivotal Tracker so that people can know you're handling this.

Reply:
Also can you explain to me the reason behind this change, I'm not so familiar with it

Reply:
In the previous implementation, we had to create the JWT classes and configure for ourselves, but when using djangorestframework-jwt, it comes with inbuilt classes.

Reply:
Ah i see, Cool, fix the merge conflicts before I can merge this please